main.go

```diff
@@ -73,6 +73,7 @@ const (
 var (
 	// ldflags
 	version string
+	dirty   string
```

Comment:
```suggestion
	dirty   bool
```

Reply:
The reason why this is set to `string` is that ldflags cannot take bool .